Index: ibmtpm/src/LibSupport.h
===================================================================
--- ibmtpm.orig/src/LibSupport.h
+++ ibmtpm/src/LibSupport.h
@@ -68,7 +68,7 @@
 #   if defined(__x86_64__) || defined(__x86_64)				\
     || defined(__amd64__) || defined(__amd64) || defined(_WIN64) || defined(_M_X64) \
     || defined(_M_ARM64) || defined(__aarch64__) \
-    || defined(__powerpc64__) || defined(__ppc64__)
+    || defined(__powerpc64__) || defined(__ppc64__) || defined(__s390x__)
 #       define RADIX_BITS                      64
 #   elif defined(__i386__) || defined(__i386) || defined(i386)		\
     || defined(_WIN32) || defined(_M_IX86)				\
